cmd/boulder-observer/main.go

```diff
@@ -33,6 +34,13 @@ func main() {
 
 	// Start the `Observer` daemon.
 	observer.Start()
+
+	go cmd.CatchSignals(log.Get(), func() {
+		observer.Stop()
+	})
+
+	// Run forever
+	select {}
```

Comment:
If you leave the `select {}` inside `observer.Start()`, then this main() could be even more similar to our other mains:
```suggestion
	go cmd.CatchSignals(log.Get(), func() {
		observer.Stop()
	})

	cmd.FailOnError(observer.Start(), "Boulder Observer daemon failed")
```